tests/page/page-mouse.spec.ts

```diff
@@ -78,6 +78,43 @@ it('should dblclick the div', async ({ page, server }) => {
   expect(event.button).toBe(0);
 });
 
+it('should mousedown the div with extra buttons', async ({ page, server }) => {
+  await page.setContent(`<div style='width: 100px; height: 100px;'>Click me</div>`);
+  await page.evaluate(() => {
+    window['mousedownPromise'] = new Promise(resolve => {
+      document.querySelector('div').addEventListener('mousedown', event => {
+        if (event.button !== 1)
+          return;
+
+        resolve({
+          type: event.type,
+          detail: event.detail,
+          clientX: event.clientX,
+          clientY: event.clientY,
+          isTrusted: event.isTrusted,
+          button: event.button,
+          buttons: event.buttons,
+        });
+      });
+    });
+  });
+  await page.mouse.move(50, 60);
+  await page.mouse.down({
+    button: 'right'
+  });
+  await page.mouse.down({
```

Comment:
i'm trying to simulate this behavior in the real browser and am failing - I don't see the second event being dispatched, how real is this scenario?

Reply:
I’ve successfully reproduced this with a mouse:
- press right mouse button without releasing it -> context menu gets open
- press middle mouse button -> this closes the context menu
- press middle mouse button again -> this fires the event that is being tested here

The scenario might not be super realistic or useful (but possible!). It’s just a one that i’ve been sort-of debugging since at first I’ve configured buttons property in a wrong way for this command (when working with Cypress and `cypress-real-events`). So I’ve just reused this exact situation for the test case here, as I was sure how it works. However, I think the same idea applies for mixing other mouse buttons together. So maybe this could be converted to a „simpler” test - with a more common mouse button combination.